assets/src/apps/delivery/store/features/activities/slice.ts

```diff
@@ -58,6 +58,23 @@ const slice: Slice<ActivitiesState> = createSlice({
       state.currentActivityId = action.payload.activityId;
     },
   },
+
+  extraReducers: (builder) => {
+    builder.addCase(loadPageState, (state, action) => {
```

Comment:
hmm why is this coming back? I think this will make a flicker on resume because it will pick the first item then the navigation will send to the previous item